### PR TITLE
Stairs issue

### PR DESCRIPTION
**Pull request by xesf**
Fixed stairs collision issue
Adding a dirty fix for LBA1 Twinsen's House Door which seem to have the wrong brick shape assigned. Will require further investigation post event.

**Preview here:** https://pr-629.lba2remake.net

### CODE REVIEW

File: src/game/scenery/isometric/IsoSceneryPhysics.ts [hunk 1]
```diff
@@ -181,7 +181,16 @@ export default class IsoSceneryPhysics {
...
-                const minY = i > 0 ? bb.min.y : -Infinity;
+                let minY = i > 0 ? bb.min.y : -Infinity;
+                if (column.shape >= 2 && column.shape <= 5) {
+                    minY = -Infinity;
+                }
+                // dirty fix for LBA1 Twinsen's house
+                // of the bricks seem to have the wrong shape attached
+                // requires more investigation
+                if (isLBA1 && scene.index === 4) {
+                    minY = i > 0 ? bb.min.y : -Infinity;
```
Comment (agrande) [inline src/game/scenery/isometric/IsoSceneryPhysics.ts:192]: Won't this affect other parts of the scene?

**Review by slwilliams — COMMENTED** [review]
This seems to have fully broken the stairs in some cases: https://pr-629.lba2remake.net/#scene=177

**Comment by xesf** [comment]
Just managed to fix it without the LBA1 dirty fix and it works for the broken 177 scene.

**Comment by agrande** [comment]
Yep, seems all good now.
Great!

**Review by agrande — APPROVED** [review]

**Pull request merged**